src/docfx/lib/json/YamlUtility.cs

```diff
@@ -62,7 +59,7 @@ public static string ReadHeader(string yaml)
         /// </summary>
         public static T Deserialize<T>(string input)
         {
-            var token = ParseAsJToken(input, file: null);
+            var (_, token) = ParseAsJToken(input, file: null);
```

Comment:
why `RemoveNulls` is not applied here?

Reply:
Same as `JsonUtility.Deserialize` not removing nulls